src/Checkbox/Checkbox.svelte

```diff
@@ -3,12 +3,21 @@
    * @event {boolean} check
    */
 
-  /** Specify the value of the checkbox */
+  /**
+   * Specify the value of the checkbox
+   * @type {string | any}
```

Comment:
Nice catch, typing this as any. We can remove the redundant "string" type.

```suggestion
   * @type {any}
```

Reply:
Absolutely. Done.